Configurations/10-main.conf

```diff
@@ -521,6 +584,10 @@ my %targets = (
     # HP/UX IA-64 targets
     "hpux-ia64-cc" => {
         inherit_from     => [ "BASE_unix", asm("ia64_asm") ],
+        policy           => 'secondary',
+        supporters       => [ 'appro@openssl.org',
+                              'Paul Dale <paul.dale@oracle.com>' ],
```

Comment:
Do we want to leave Andy's address on these if he only wants to be on the hook for "community"-level support?